content/docs/object-reference/build/requirements.md

```diff
@@ -0,0 +1,42 @@
+# Requirements
```

Comment:
This pages should be in `user-guide`, not `object-reference`. We decided to keep in `user-guide` human-readable part of these pages, while keep in `object-reference` auto generated parts + some brief descriptions / explanations aimed for developers / advanced users.

Reply:
I am not sure, because even for `Kubernetes` and `Sagemaker`, they are in `object-reference`.

Check out https://github.com/iterative/mlem.ai/pull/183 for example.

Or should I change both PRs? i.e. this as well as kubernetes docs both?

Reply:
ok, moved to user-guide, will move Kubernetes docs too